### PR TITLE
New 'Listening...' Placeholder Translation and Display Mic Icon Only on Supported Browsers

### PR DESCRIPTION
**Pull request by prem-k-r**
New element "Listening..." translation

Showing the mic icon only in Chrome and Edge

### CODE REVIEW

**Comment by prem-k-r** [comment]
@itz-rj-here need to change? I'm not sure

@ceskyDJ new text, to be translated

**Review by itz-rj-here — APPROVED** [review]
Its ok.

**Comment by prem-k-r** [comment]
ok

**Comment by ceskyDJ** [comment]
I'll check it in a few dozens of minutes.

**Comment by prem-k-r** [comment]
@XengShi 
As a user, I think it would be better if there were no transition on hover.
The animation should start on click, making it easier to track when the webpage is accessing audio.

File: script.js [hunk 1]
```diff
@@ -644,74 +644,91 @@ document.addEventListener("DOMContentLoaded", () => {
 
 
 //  -----------Voice Search------------
-const micIcon = document.getElementById("micIcon");
```
Comment (prem-k-r) [inline script.js:647]: select "Hide Whitespace" when reviewing.

**Comment by XengShi** [comment]
> @XengShi As a user, I think it would be better if there were no transition on hover. The animation should start on click, making it easier to track when the webpage is accessing audio.

Okay, will remove that

**Comment by ceskyDJ** [comment]
Hmm, the feature doesn't work for me. I click on the microphone icon, which shows "Listening..." for a few seconds and goes back to "Start typing..." afterward. It required browser (Brave) permission, so I accepted it. It doesn't work anyway.

**Comment by ceskyDJ** [comment]
Czech translation:

- "Type here..." --> "Začněte psát…" (or maybe something better, but I don't have a better idea now)
- "Listening..." --> "Poslouchám…"

**Comment by ceskyDJ** [comment]
Oh, it's already merged :D. I'll create a new pull request for it.

**Comment by itz-rj-here** [comment]
@prem-k-r

**Comment by ceskyDJ** [comment]
Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.

**Comment by prem-k-r** [comment]
> Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.

I understand. Maybe we can go with **"Type your query here..."**
XengShi's call

**Comment by itz-rj-here** [comment]
> > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> 
> I understand. Maybe we can go with **"Type your query here..."** XengShi's call

Bruh someone tried to fake confess me on instagram now -_-

**Comment by prem-k-r** [comment]
> Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.

@XengShi does it worked for you for Brave?

Used user agent approach to identify browser

**Comment by XengShi** [comment]
> > > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> > 
> > I understand. Maybe we can go with **"Type your query here..."** XengShi's call
> 
> Bruh someone tried to fake confess me on instagram now -_-

Whatttr?

**Comment by itz-rj-here** [comment]
> > > > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> > > 
> > > 
> > > I understand. Maybe we can go with **"Type your query here..."** XengShi's call
> > 
> > 
> > Bruh someone tried to fake confess me on instagram now -_-
> 
> Whatttr?

Uhhhhhhhh forget that. -_-

**Comment by ceskyDJ** [comment]
> icon was visible in Brave? included changes from this PR?

What icon? I'm on `main`'s latest commit, so yeah.

**Comment by ceskyDJ** [comment]
> > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> 
> I understand. Maybe we can go with **"Type your query here..."** XengShi's call

Could you please let me know, when this is decided? Btw, "Type your query here..." has the same problem, as I noted before.

**Comment by prem-k-r** [comment]
> > icon was visible in Brave? included changes from this PR?
> 
> What icon? I'm on `main`'s latest commit, so yeah. I see Brave's icon signalizing that the webpage is listening for my input, but it's just for a few seconds, nothing more. The microphone icon in the UI of this extension is visible, of course.

ah sorry, I meant Mic icon.
Are you using any other custom user agent? can you please share your user agent
image

**Comment by ceskyDJ** [comment]
> > > icon was visible in Brave? included changes from this PR?
> > 
> > 
> > What icon? I'm on `main`'s latest commit, so yeah. I see Brave's icon signalizing that the webpage is listening for my input, but it's just for a few seconds, nothing more. The microphone icon in the UI of this extension is visible, of course.
> 
> ah sorry, I meant Mic icon. Are you using any other custom user agent? can you please share your user agent image

Here it's:
```
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
```

**Comment by ceskyDJ** [comment]
> > > > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> > > 
> > > 
> > > I understand. Maybe we can go with **"Type your query here..."** XengShi's call
> > 
> > 
> > Could you please let me know, when this is decided? Btw, "Type your query here..." has the same problem, as I noted before.
> 
> then `Search the web...`

Sounds great to me. @XengShi, what do you think?

**Comment by XengShi** [comment]
> > > > Before that… @prem-k-r, "Type here..." sounds like "baby instructions" (I hope you get it). I'd prefer something like "Enter your query..." or "Type your query" or something like that – everyone knows where they should type the query :D.
> > > 
> > > 
> > > I understand. Maybe we can go with **"Type your query here..."** XengShi's call
> > 
> > Could you please let me know, when this is decided? Btw, "Type your query here..." has the same problem, as I noted before.
> 
> then `Search the web...`

"Browse the world",
"Search with [selected search engine]",
"Search the web",
"Start typing...."
"Explore the web"

How about showing this kinda message randomly 😛

**Pull request merged**